backend/typescript/rest/tagRoutes.ts

```diff
@@ -27,12 +27,12 @@ tagRouter.get(
 );
 
 tagRouter.delete(
-  "/:id",
+  "/:name",
```

Comment:
I don't think we actually need a delete route on Tags since they're deleted automatically with reviews :thinking:

Reply:
Should we keep it around just in case something wonky happens and we need to manually delete a tag?

Reply:
I think this route (and the `deleteTag` function in the TagService) would only work if the Tag to be deleted is not associated with any books (in which case the Review Service would have already implicitly deleted it). I guess there's no harm in leaving it tho. If you're gonna do so, maybe add a comment noting it's only for dev purposes

Reply:
On second thought, it doesn't make much sense to have it, so it is now removed.